### ISSUE TITLE
Plugin Support

### ISSUE THREAD

**Issue by danrossi** [sampotts/plyr#1242]
I see some communication regards to plugin support but I see nothing in here regards to this. Is there a way to easily implement loading plugins. External from the player ? 

The plugin loading needs to happen in the constructor before loading happens.

**Comment by sampotts** [comment]
It's something I'd like to build in but haven't spent much time thinking about how it could be structured. Any help would be appreciated. Making the YouTube and Vimeo players into true plugins would be a good start.

**Comment by danrossi** [comment]
from my experience with flowplayer / video.js. This is a very basic 
plugin registry. They are called before a source is set.  It seems to 
work for me.

I haven't looked further because it lacks HLS / Dash. I was looking to 
port my plugins over but it needs alot of work with the player first.

https://github.com/danrossi/plyr/commit/4a0900bf1821191e4342906a5cae61c457367bb9

------ Original Message ------

**Comment by filips123** [comment]
There should also be support for custom themes.

**Comment by sampotts** [comment]
> There should also be support for custom themes.

There already is via CSS.

**Comment by filips123** [comment]
@sampotts Yes but if plugins will be supported, it will be easier to change it.
